libsolidity/ast/AST.h

```diff
@@ -557,13 +557,34 @@ class ContractDefinition: public Declaration, public StructurallyDocumented, pub
 	util::LazyInit<std::vector<EventDefinition const*>> m_interfaceEvents;
 };
 
+/**
+ * A sequence of identifiers separated by dots
+ */
```

Comment:
Maybe add: Used outside the expression context. Inside the expression context, this is a sequence of Identifier and MemberAccess.